doc/nrf/known_issues.rst

```diff
@@ -1210,6 +1210,9 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 SoftDevice Controller
 =====================
 
+When working with the SoftDevice Controller, you should be aware of the following known issues.
+In addition, see :ref:`softdevice_controller_limitations` for permanent limitations.
```

Comment:
I think this would read better as one sentence:
```suggestion
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
```